bluebutton-data-model-rif/src/main/resources/db/migration/V11__Create_medicare_beneficiary_id_history_table.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * Creates the MedicareBeneficiaryIdHistory table required by the application. This
+ * schema is derived from the JPA metadata, via the 
+ * gov.hhs.cms.bluebutton.data.pipeline.rif.schema.HibernateSchemaPrinter 
+ * utility.
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "MedicareBeneficiaryIdHistory" (
+  "medicareBeneficiaryIdKey" numeric not null,
```

Comment:
Is this going to be an auto-generated sequence, or is it coming from the RIF? If it's auto-generated, you'll need a `create sequence` similar to what we have in https://github.com/CMSgov/bluebutton-data-model/blob/master/bluebutton-data-model-rif/src/main/resources/db/migration/V9__Create_beneficiary_history_table.sql

Reply:
This id key is coming from RIF.